src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -195,4 +259,18 @@ const styles = StyleSheet.create({
   }
 });
 
+TopicFragment.propTypes = {
+  isLoadingDiscoveryTopic: PropTypes.bool,
+  followedTopic: PropTypes.array,
+  unfollowedTopic: PropTypes.array,
+  isFirstTimeOpen: PropTypes.bool,
+  setIsFirstTimeOpen: PropTypes.func,
+  setSearchText: PropTypes.func,
+  setFollowedTopic: PropTypes.func,
+  setUnfollowedTopic: PropTypes.func,
+  fetchData: PropTypes.func,
+  searchText: PropTypes.string,
+  withoutRecent: PropTypes.bool
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

PropTypes are added to the component. This is a good practice as it helps with type checking and can prevent bugs related to incorrect data types.
```diff
+ TopicFragment.propTypes = {
+   isLoadingDiscoveryTopic: PropTypes.bool,
+   followedTopic: PropTypes.array,
+   unfollowedTopic: PropTypes.array,
+   isFirstTimeOpen: PropTypes.bool,
+   setIsFirstTimeOpen: PropTypes.func,
+   setSearchText: PropTypes.func,
+   setFollowedTopic: PropTypes.func,
+   setUnfollowedTopic: PropTypes.func,
+   fetchData: PropTypes.func,
+   searchText: PropTypes.string,
+   withoutRecent: PropTypes.bool
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->